app/src/main/res/values/strings.xml

```diff
@@ -716,6 +716,8 @@
     <string name="snackbar_tabs_closed">Tabs closed</string>
     <!-- Text shown in snackbar when user adds a site to home fragment. Placeholder replaced with app name -->
     <string name="snackbar_added_to_firefox_home">Added to %s home!</string>
+    <!-- Text shown in snackbar when user adds a site to top sites -->
+    <string name="snackbar_added_to_top_sites">Added to top sites!</string>
```

Comment:
I think we need to change this to "Added to top sites" to match the specs mentioned in issue. (We need to remove the "!")

Reply:
@topotropic Should we keep or remove the exclamation sign ("!") ?. In the original mock the confirmation message had one. TY!

Reply:
I spoke with @topotropic and she said let's keep the exclamation mark.